api/src/main/resources/application.properties

```diff
@@ -15,6 +15,8 @@ api.okta-jwt-audience=${AB2D_OKTA_JWT_AUDIENCE:#{'AB2D'}}
 api.okta-connection-timeout=${AB2D_OKTA_CONNECTION_TIMEOUT:#{5}}
 api.okta-read-timeout=${AB2D_OKTA_READ_TIMEOUT:#{5}}
 
+api.requestlogging.filter=^/health$
```

Comment:
Let's add a note here that adding a comma will cause the filters to be split, just in case someone tries to add a comma in a regex itself.

Reply:
commented and pushed